src/plan/trace.rs

```diff
@@ -1,6 +1,9 @@
 use crate::util::queue::SharedQueue;
 use crate::util::{Address, ObjectReference};
 
+/// This trait describes the core functionality for a transitive
+/// closure over the heap. Any class implementign this trait holds the global state, TraceLocal
```

Comment:
I don't understand the last sentence here.
Does the "," need to be changed to "."?
Probably rewriting it as:

Any class implementing this trait holds the global state (per-thread state is handled by TraceLocal and its implementers).

Reply:
Yeah. That should be a period `.`